DOCUMENTATION.md

```diff
@@ -340,15 +340,19 @@ The personalisation argument always contains the following required parameters f
 
 - `address_line_1`
 - `address_line_2`
-- `postcode` (this needs to be a real UK postcode)
+– `address_line_7` this needs to be a real UK postcode or the name of a country outside the UK
+
+Notify uses `address_line_7` to check for international addresses. This is so we can charge you the correct postage.
```

Comment:
Is it useful to have this context?

Reply:
yes but no - we don't just look at address line 7. We look at the very last non-empty line. For example, consider the following address:

address_line_1: "123 champs elysees"
address_line_2: "paris"
address_line_3: "12345"
address_line_4: "FRANCE"
address_line_5: ""
address_line_6: ""
address_line_7: ""

we'd check that the last non empty line has a correct country or UK postcode

Reply:
Cool. So we *can* still read that as a valid international address, but we want to encourage users to use line_7?